core/src/main/java/com/linecorp/armeria/common/HttpHeadersBase.java

```diff
@@ -484,6 +498,16 @@ public long contentLength() {
         }
     }
 
+    public void contentLengthUnknown(boolean contentLengthUnknown) {
+        remove(HttpHeaderNames.CONTENT_LENGTH);
```

Comment:
Question) Do we remove `CONTENT_LENGTH` even if `contentLengthUnknown = false`?

Reply:
It sounds like a bug. 😅